Bixal/move.mil
# Move6 add captcha headers

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Add site and secret key as env variables
- Adds google header to ngnix config

## Testing

To verify the changes proposed in this pull request…

1. Pull changes
1. execute `make composer` and `make cim`. Then clear caches.
1. Add your keys in your .env file
1. You should be able to see a recaptcha frame on register and reset password form. 
1. Also, if you run `docker-compose run --rm php drush config-get recaptcha.settings --include-overridden` you should see the keys you have on your .env file.
